### ISSUE TITLE
"Sumo" is confusing and unnecessary

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11481]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#1331|
|Type|defect|
|Created|2005-11-13 11:26:07Z|

<details><summary>Searchable metadata</summary>

```
trac-id__1331 1331
type__defect defect
reporter__exarkun exarkun
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1131881167000000 1131881167000000
changetime__1142816104000000 1142816104000000
version__ 
owner__ 
cc__glyph cc__radix cc__exarkun cc__spiv cc__itamarst cc__jml
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun commented|
|-|-|

```
#!html
<pre>
The current "Sumo" package *is* Twisted.  All of it.  The name should reflect
this fact by including "Twisted" and a versio number, and nothing else.

The Core package should be de-emphasized.  It's a specialty package for people
who already know about Twisted and who know they need just it.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)|@itamarst commented|
|-|-|

```
#!html
<pre>
Also, I suggest the newly renamed package should not include deprecated
subpackages like flow and pair.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph commented|
|-|-|

```
#!html
<pre>
Agreed.

Considering the lack of maintenance on both packages, I think they should
probably be moved off of tm.com entirely; I haven't seen any maintenance from
cce (although I believe he uses flow himself) and I have seen Tv quoted as
saying that pair was split out to "give it a convenient place to die".

Maintenance can be picked up by their respective authors on different sites, and
we can provide links.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun commented|
|-|-|

```
#!html
<pre>
Twisted internet project page has been updated.  It now suggests the sumo
package over the core-only package.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph commented|
|-|-|

```
#!html
<pre>
Awesome, thanks exarkun.  Now maybe people will stop downloading the wrong thing!

When the next release goes out, the tarballs should be renamed
"TwistedCore-2.2.0" and "Twisted-2.2.0".
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix set owner to @exarkun|
|-|-|

tarball rename in [branches/tarball-rename-1331](branches/tarball-rename-1331)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @radix|
|-|-|

Some more text about the "a.b.x" part of the version would be good.  I gather this will start with whatever comes after 2.2.0 and go from there.  What is the current TwistedCore version?

Other than this clarification, I think everything in the branch looks good to merge.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix set status to `closed`|
|-|-|

(In [16361]) Rejigger the names of the release tarballs to better reflect sanity - Fixes [#11481](https://github.com/twisted/twisted/issues/11481).

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
